pulsar-client-cpp/lib/ClientConnection.cc

```diff
@@ -119,6 +119,9 @@ static Result getResult(ServerError serverError) {
 }
 
 static bool file_exists(const std::string& path) {
+    if(path.empty()){
```

Comment:
```suggestion
    if (path.empty()){
```

The format checker is failing on the missing space.